src/routing.py

```diff
@@ -179,6 +183,50 @@ def _init_route_info(self):
         self.route_props.paths_matrix = list()
         self.route_props.mod_formats_matrix = list()
         self.route_props.weights_list = list()
+        self.route_props.connection_index = list()
+        self.route_props.path_index = list()
+
+    def load_k_shortest(self):
+        """
+        Load the k-shortest paths from an external file.
+        """
+        loaded_data_dict = np.load('USB6014-10SP.npy', allow_pickle=True)
+        src_des_list = [int(self.sdn_props.source), int(self.sdn_props.destination)]
+        
+        path_cnt = 0
+        for item in loaded_data_dict:
+            cnt = 0
+            first_element = item[5][0][0][0][0]
```

Comment:
Is first element equal to the first node or first path?

Reply:
node